allennlp/models/bidaf.py

```diff
@@ -331,18 +330,12 @@ def predict_span(self, question: TextField, passage: TextField) -> Dict[str, Any
                                                for_training=False)
         output_dict = self.forward(**model_input)
 
-        # Remove batch dimension, as we only had one input.
-        span_start_logits = output_dict["span_start_logits"]
-        span_end_logits = output_dict["span_end_logits"]
-        best_span = self._get_best_span(span_start_logits, span_end_logits)
-        best_span = tuple(best_span.data.squeeze(0).numpy())
-        best_span_str = ' '.join(passage.tokens[best_span[0]:best_span[1]])
-
+        # Here we're just removing the batch dimension and converting things to numpy arrays /
+        # tuples instead of pytorch variables.
         return {
                 "span_start_probs": output_dict["span_start_probs"].data.squeeze(0).numpy(),
```

Comment:
Do these need `.cpu()` to allow prediction on the GPU?

Reply:
Ah, yes, good catch, thanks.